cmd/laas/main.go

```diff
@@ -68,9 +72,14 @@ func main() {
 
 	r := gin.Default()
 	r.NoRoute(api.HandleInvalidUrl)
-	r.GET("/api/licenses", api.GetAllLicense)
-	r.GET("/api/license/:shortname", api.GetLicense)
-	r.POST("/api/license", api.CreateLicense)
-	r.PATCH("/api/license/:shortname", api.UpdateLicense)
+	authorized := r.Group("/")
+	authorized.Use(auth.AuthenticationMiddleware())
+	authorized.GET("/api/license/:shortname", api.GetLicense)
```

Comment:
Why is getting a license requires authorization?

Reply:
Ummm... There the endpoint to list all users and specific user is unauthenticated.

Endpoints to Create/modify licenses and every endpoint related to users should be authenticated.